pkg/utils/constants.go

```diff
@@ -92,10 +93,11 @@ const (
 	MissingIeee8021xConfiguration     ReturnCode = 117
 
 	// (150-199) Maintenance Errors
-	SyncClockFailed      ReturnCode = 150
-	SyncHostnameFailed   ReturnCode = 151
-	SyncIpFailed         ReturnCode = 152
-	ChangePasswordFailed ReturnCode = 153
+	SyncClockFailed        ReturnCode = 150
+	SyncHostnameFailed     ReturnCode = 151
+	SyncIpFailed           ReturnCode = 152
+	ChangePasswordFailed   ReturnCode = 153
+	UpdateDeviceInfoFailed ReturnCode = 154
```

Comment:
why not `SyncDeviceInfoFailed` like the others?

Reply:
fixed